remotecommanddispatcher/src/main/java/com/tealium/remotecommanddispatcher/RemoteCommandsManager.kt

```diff
@@ -0,0 +1,58 @@
+package com.tealium.remotecommanddispatcher
+
+import com.tealium.core.TealiumConfig
+import com.tealium.remotecommanddispatcher.remotecommands.JsonRemoteCommand
+import com.tealium.remotecommands.RemoteCommand
+
+interface CommandsManager {
+    fun add(remoteCommand: RemoteCommand, filename: String? = null, remoteUrl: String? = null)
+    fun remove(commandId: String)
+    fun removeAll()
+    fun getRemoteCommand(commandId: String): RemoteCommand?
+    fun getJsonRemoteCommandConfig(commandId: String): RemoteCommandConfigRetriever?
+    fun getJsonRemoteCommands(): List<RemoteCommand>
```

Comment:
Similar comment as above. The method is called `getJsonRemoteCommands` but you return a `List<RemoteCommand>`.